removed --init step and added step for adding new nodes

jseldess opened the pull request:
Fixes #30 

- Removed `--init` step from **Secure Mode** instructions.
- Added step on adding nodes to both the **Dev Mode** and **Secure Mode** instructions. 
- Tweaked the intro to mention adding additional nodes.

@spencerkimball, @mberhault, please let me know what you think. Instead of adding a third section for a "multi-node local cluster", I just added a step to each of the existing procedures. Happy to try another approach, if you prefer. 

@spencerkimball: I've used `localhost` in the secure example, even though it's currently a problem: [#4118](https://github.com/cockroachdb/cockroach/issues/4118) and [#3358](https://github.com/cockroachdb/cockroach/issues/3358).

In case you'd rather review an html version: http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/start-a-local-cluster.html

vivekmenezes commented on start-a-local-cluster.md:
````diff
@@ -32,34 +34,36 @@ Data is stored on-disk and client/server communication is secure. Setup involves
+    
     ```bash
-    $ ./cockroach init --stores=ssd=dev/data
+    $ ./cockroach start --dev pgport=15433 --port=26258 --join=localhost:26257
````
--pgport

the same below

vivekmenezes commented:
On Wed, Feb 3, 2016 at 6:00 PM jseldess <notifications@github.com> wrote:

> Fixes #30 <https://github.com/cockroachdb/docs/issues/30>
>
>    - Removed --init step from *Secure Mode* instructions.
>    - Added step on adding nodes to both the *Dev Mode* and *Secure Mode**
>    instructions.
>    - Tweaked the intro to mention adding additional nodes.
>
> @spencerkimball <https://github.com/spencerkimball>, @mberhault
> <https://github.com/mberhault>, please let me know what you think.
> Instead of adding a third section for a "multi-node local cluster", I just
> added a step to each of the existing procedures. Happy to try another
> approach, if you prefer.
>
> @spencerkimball <https://github.com/spencerkimball>: I've used localhost
> in the secure example, even though it's currently a problem: #4118
> <https://github.com/cockroachdb/cockroach/issues/4118> and #3358
> <https://github.com/cockroachdb/cockroach/issues/3358>.
>
>
I think if it's still a problem we shouldn't be checking this in without
fixing those issues.


> In case you'd rather review an html version:
> http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/start-a-local-cluster.html
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/cockroachdb/docs/pull/31
> Commit Summary
>
>    - removed --init step and added step for adding new nodes
>
> File Changes
>
>    - *M* start-a-local-cluster.md
>    <https://github.com/cockroachdb/docs/pull/31/files#diff-0> (52)
>
> Patch Links:
>
>    - https://github.com/cockroachdb/docs/pull/31.patch
>    - https://github.com/cockroachdb/docs/pull/31.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/docs/pull/31>.
>

jseldess commented:
Thanks, @vivekmenezes. Just corrected those missing dashes and, for now, removed usage of `localhost`. Also rephrased explanations of flags a bit. Please let me know if you have any other thoughts/concerns.

spencerkimball commented on start-a-local-cluster.md:
````diff
@@ -32,34 +34,38 @@ Data is stored on-disk and client/server communication is secure. Setup involves
+    
     ```bash
-    $ ./cockroach init --stores=ssd=dev/data
+    $ ./cockroach start --dev --port=26258 --pgport=15433 --join=<your local host>:26257
````
This doesn't currently work – you're only creating two separate dev mode clusters. They're not talking to each other. I don't think we should support using --dev mode in this way.

I'm going to submit some fixes for this.

You should go back to have a Standard mode like we used to have.

jseldess replied:
Thanks for catching that, @spencerkimball. Changed **Dev Mode** back to single-node only, changed **Secure Mode** back to **Standard Mode**, and tweaked intro a bit.

Let me know if you see anything else, or if we're good to merge.

jseldess commented:
Thanks for catching that, @spencerkimball. Changed Dev Mode back to single-node only, changed Secure Mode back to Standard Mode, and tweaked intro a bit.

Let me know if you see anything else, or if we're good to merge.

spencerkimball commented:
LGTM
